streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -278,10 +276,9 @@ public void onPartitionsAssigned(final Collection<TopicPartition> assignment) {
 
         @Override
         public void onPartitionsRevoked(final Collection<TopicPartition> assignment) {
-            log.debug("{} at state {}: partitions {} revoked at the beginning of consumer rebalance.\n" +
+            log.debug("at state {}: partitions {} revoked at the beginning of consumer rebalance.\n" +
```

Comment:
same here capitalize

Reply:
`completed` is in lower case because before this we have applied a `logPrefix` using `LogContext`